R/002_plot_components.R

```diff
@@ -49,7 +60,24 @@ create_rawplot_components <- function(proportional,
   return(plot_component)
 }
 
-# Function for creation of list of TRUE/FALSE for delta plot components that will be built
+#' Generates list of TRUE/FALSE for delta plot components that will be built
+#' 
+#' This function generates a list of booleans to be passed into the delta plot function.
```

Comment:
Same as above but can you reword this to "generates a list of booleans determining whether certain plot components will be constructed for the deltaplot."

Reply:
done